db/migrate/20210819170858_create_screenings.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CreateScreenings < ActiveRecord::Migration[6.1]
+  def change
+    create_table :screenings do |t|
+      t.datetime :date_and_time, null: false
```

Comment:
Maybe more readable would be `starts_at` instead of `date_and_time`?:)

Reply:
I wanted this to explicitly state screening requires starting date and time, but now I see how unclear this may be. I'll come up with something more descriptive, i.e. _starting_ _date_and_time. :)